packages/code-studio/jestBabelTransform.js

```diff
@@ -0,0 +1,32 @@
+// This was created from ejecting a blank CRA-4 project w/ TypeScript support
+// Code-studio tests do not run properly without using this transform
```

Comment:
Any more detail on why they don't run without this transform? I'm a little confused about this part.

Reply:
Fixed to use babel-jest transform just for code-studio instead of ts-jest. This removes the need for the transform from CRA